docker-compose.dev-staging.yaml

```diff
@@ -0,0 +1,33 @@
+######################################################################
+# Docker compose override file for development + staging environments
+######################################################################
+services:
+  postgres:
+    image: postgres:17                      # official Postgres 17 image
+    container_name: postgres                # fixed name for easier links
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB}
+      POSTGRES_PORT: ${POSTGRES_PORT}
+      POSTGRES_OPTIONS: ${POSTGRES_OPTIONS}
+    restart: unless-stopped                 # auto-restart unless stopped
+    ports:
+      - "${POSTGRES_PORT}:5432"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data  # persist data
+    networks:
+      - backend_network
+    healthcheck:                            # wait until DB is ready
+      test: ["CMD-SHELL", "pg_isready -U ${POSTGRES_USER} -d ${POSTGRES_DB}"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+
+  migrator:
```

Comment:
Without knowing much about how docker-compose handles these, do we need:
```yaml
    restart: "no"                           # run once, then exit
    command: []  
```
and any others here?

Reply:
Great question, I'll double check the migrations by starting fresh on our production DB.

Reply:
@calebbourg it does not seem necessary. The migrator service came up, did it's thing, and spun down when migrationctl finished.